submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/SubmissionService.kt

```diff
@@ -58,6 +60,8 @@ class SubmissionService(
     @RabbitListener(queues = [SUBMISSION_REQUEST_QUEUE], concurrency = "1-1")
     fun processSubmission(request: SubmissionRequestMessage) {
         logger.info { "received process message for submission ${request.submission}" }
+        Thread.sleep(30_000L)
```

Comment:
why are we adding this?

Reply:
It was for testing purposes. I'll remove it, nice catch.